swiftMongoDB/swiftMongoDB-Bridging-Header.h

```diff
@@ -0,0 +1,6 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
+#import <bson.h>
```

Comment:
You don't need a bridging header with frameworks - SwiftMongoDB.h should be enough :)